code/controllers/subsystems/ticker.dm

```diff
@@ -444,16 +444,24 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	for(var/s in SSghostroles.spawners)
 		var/datum/ghostspawner/G = SSghostroles.spawners[s]
-		if(G.enabled && !("Antagonist" in G.tags) && !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)))
+		if(G.enabled \
+			&& !("Antagonist" in G.tags) \
+			&& !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)) \
+			&& (("Simple Mobs" in G.tags) || ("External" in G.tags)) \
+		)
```

Comment:
Generally alright, but a new variable that controls if it should be visible in the lobby might be the better approach.

Reply:
changed from `&& (("Simple Mobs" in G.tags) || ("External" in G.tags))`
into `&& (G.req_perms == null)`

Should be better like this.
I'd rather not be adding yet another variable.